assembl/static2/css/components/socialShare.scss

```diff
@@ -12,16 +12,28 @@
   padding-bottom: 20px;
 }
 
-*:focus {
+.modal-content
```

Comment:
it will modify all modals, no?

Reply:
this was put to deal with an undesirable highlighting of divs (different depending on the browser), would we ever want such a thing to happen in a modal ? even anywhere else... this is an annoying behaviour

Reply:
no problem, it was more a remark/"warning"